pkg/features/kcp_features.go

```diff
@@ -97,7 +97,7 @@ func (f *kcpFeatureGate) Type() string {
 // here. The features will be available throughout Kubernetes binaries.
 var defaultGenericControlPlaneFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	LocationAPI:  {Default: true, PreRelease: featuregate.Alpha},
-	SyncerTunnel: {Default: false, PreRelease: featuregate.Alpha},
+	SyncerTunnel: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
all security is in-place?

Reply:
No, you're right, after discussing it, we still need to implement https://github.com/kcp-dev/kcp/issues/1975.
I'll remove the feature flag switch commit from this PR

Reply:
Done